_includes/misc/purchase-premium-plugins.md

```diff
@@ -0,0 +1,51 @@
+{% if extensionType != 'Packs' %}
+{% assign extensionType = 'plugin' %}
+{% endif %}
+{% if pluralExtensionType == true %}
+{% assign isAre = 'are' %}
+{% else %}
+{% assign isAre = 'is' %}
+{% endif %}
+### Getting started with TinyMCE {{pluginname}}
+
+#### Creating an account
+
+To try out the **{{pluginname}} {{extensionType}}** and the {{site.cloudname}}, create a free [{{site.accountpage}}]({{site.accountsignup}}). Sign-up for a free {{site.accountpage}} and receive a 30-day trial of the {{site.companyname}} [premium plugins]({{site.plugindirectory}}), skins, and icon packs; with no credit card information or commitment required.
+
+#### Buy TinyMCE {{pluginname}}
+
+{% if pluginminimumplan == 'tierone' %}
+The **{{pluginname}} {{extensionType}}** {{isAre}} included in the following {{site.cloudname}} plans:
+
+- The [{{site.tieroneplan}}]({{site.pricingpage}}).
+- The [{{site.tiertwoplan}}]({{site.pricingpage}}).
+- The [{{site.tierthreeplan}}]({{site.pricingpage}}).
+- [{{site.enterpriseplan}}]({{site.pricingpage}}).
+
+{% elsif pluginminimumplan == 'tiertwo' %}
+The **{{pluginname}} {{extensionType}}** {{isAre}} included in the following {{site.cloudname}} plans:
+
+- The [{{site.tiertwoplan}}]({{site.pricingpage}}).
+- The [{{site.tierthreeplan}}]({{site.pricingpage}}).
+- [{{site.enterpriseplan}}]({{site.pricingpage}}).
+
+{% elsif pluginminimumplan == 'tierthree' %}
+The **{{pluginname}} {{extensionType}}** {{isAre}} included in the following {{site.cloudname}} plans:
+
+- The [{{site.tierthreeplan}}]({{site.pricingpage}}).
+- [{{site.enterpriseplan}}]({{site.pricingpage}}).
+
+{% elsif pluginminimumplan == 'enterprise' %}
+The **{{pluginname}} {{extensionType}}** {{isAre}} included in [{{site.enterpriseplan}}]({{site.pricingpage}}).
+
+{% endif %}
+
+### Further information
+
+For information on:
+
+- {{site.cloudname}} plans, see: [{{site.companyname}} Pricing Plans]({{site.pricingpage}}).{% if extensionType == 'Packs' %}
+- Enabling premium skins, see: [The `skin` setting]({{site.baseurl}}/configure/editor-appearance/#skin).
```

Comment:
Should this information only be included on the skins and icons pack pages? As it's a bit confusing to see it mentioned on the other pages when it's not related to the plugin.